contracts/strategies/extensions/allocate/IAllocationExtension.sol

```diff
@@ -6,13 +6,13 @@ interface IAllocationExtension {
     error INVALID_ALLOCATION_TIMESTAMPS();
 
     /// @dev Error thrown when trying to call the function when the allocation has started
-    error ALLOCATION_HAS_STARTED();
+    error ALLOCATION_HAS_ALREADY_STARTED();
 
     /// @dev Error thrown when trying to call the function when the allocation is not active
-    error ALLOCATION_NOT_ACTIVE();
+    error ALLOCATION_IS_NOT_ACTIVE();
 
     /// @dev Error thrown when trying to call the function when the allocation has ended
-    error ALLOCATION_NOT_ENDED();
+    error ALLOCATION_IS_NOT_ENDED();
```

Comment:
I had to make this little change in order to prevent colliding with the same errors in `Errors.sol`, maybe we should think about deprecating that library in a future PR